lsp-mode.el

```diff
@@ -3354,8 +3357,10 @@ in that particular folder."
                                   (error nil)))
    (when (featurep 'project)
      (when-let ((project (project-current)))
-       (car (with-no-warnings
-              (project-roots project)))))))
+       (if (fboundp 'project-roots)
+           (car (with-no-warnings
```

Comment:
Can you remove this `with-no-warning` and see if it works?

Reply:
yeah